security_inode_unlink() was used in filter_vfs_unlink() to avoid
lock ordering problems.  This is no longer needed for Lustre 2.4
with the change to osd-ldiskfs, but is kept for compatibility with
older server code so they can share the same kernel.

Index: linux+rh+chaos/security/security.c
===================================================================
--- linux+rh+chaos.orig/security/security.c
+++ linux+rh+chaos/security/security.c
@@ -60,6 +60,7 @@ int __init security_init(void)
 
 	return 0;
 }
+EXPORT_SYMBOL(security_inode_unlink);
 
 /* Save user chosen LSM */
 static int __init choose_lsm(char *str)
